src/App.css

```diff
@@ -52,7 +62,7 @@ code {
 .loader {
   margin: auto;
   border: 2px solid rgba(0, 123, 255, 0.15);
-  border-top: 2px solid #009999;
+  border-top: 2px solid #0099;
```

Comment:
Does this work? I had the impression that hash color values should be either 3 or  6 characters long.

Reply:
It actually works